src/components/Grants/RequestBanner.css

```diff
@@ -0,0 +1,39 @@
+.RequestBanner {
+  background: linear-gradient(89.29deg, #f6f5f8 0%, rgba(246, 245, 248, 0) 100%);
```

Comment:
Don't we have these colors in the theme?

Reply:
nope :(